internal/cli/utils_shared.go

```diff
@@ -55,7 +53,7 @@ func BuildOauthTokenParams(clientID, clientSecret, audience string) url.Values {
 }
 
 // runClientCredentialsFlow runs an M2M client credentials flow without opening a browser
-func runClientCredentialsFlow(cli *cli, c *management.Client, clientID string, audience string, tenant Tenant) (*authutil.TokenResponse, error) {
+func runClientCredentialsFlow(_ *cli, c *management.Client, clientID string, audience string, tenant Tenant) (*authutil.TokenResponse, error) {
```

Comment:
Maybe we can remove the `_ *cli` parameter?

Reply:
Fixed in https://github.com/auth0/auth0-cli/pull/489/commits/802fb15094837e7be3b28f8866430d2f9d5f887f